recipes/dynare_preprocessor/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: dynare_preprocessor
+  version: 0.0.1
```

Comment:
Please place this package into a pre-release label until it has a stable release. Users of our channel should get stable non-experimental builds by default.

https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

From the downloaded artifacts:
```
project('dynare-preprocessor', 'cpp',
        version : '6-unstable',
        default_options : [ 'cpp_std=gnu++20', 'warning_level=2' ],
        meson_version : '>=0.64.0')
```

Reply:
Thank you @carterbox for your feedback.
Just to make sure: do you make a difference between the experimental build and the experimental package ?
If I look at what upstream is doing, 5.5 is stable, but is using the old build system. Hypothetically, could a conda package eventually provide a stable release for 5.5, backporting only the new build system that is much simpler ?